squad_client/core/models.py

```diff
@@ -232,12 +232,44 @@ class TestJob(SquadObject):
 
 class TestRun(SquadObject):
 
+    def __init__(self):
+        self.__total_pass__ = 0
+        self.__total_fail__ = 0
+        self.__total_skip__ = 0
+        self.__total_xfail__ = 0
+        self.test_suites = []
+        self.metric_suites = []
+
     endpoint = '/api/testruns/'
     attrs = ['url', 'id', 'metadata_file', 'log_file',
              'created_at', 'completed', 'datetime', 'build_url',
              'job_id', 'job_status', 'job_url', 'resubmit_url', 'data_processed',
              'status_recorded', 'build', 'environment']
 
+    @property
+    def total_pass(self):
+        if self.__total_pass__ == 0 and len(self.test_suites):
+            self.__total_pass__ = len([y for x in self.test_suites for y in x.tests if y.status == 'pass'])
```

Comment:
I understand i think one letter variables are OK when there's only one. It got a bit confusing when you named them `x` and `y`. Maybe `s` and `t` ?

Reply:
@chaws this has changed. Resolving.